Source/Frontend/UI/Forms/RTC_NewBlastEditor_Form.cs

```diff
@@ -1619,43 +1583,39 @@ private void bakeROMBlastunitsToFileToolStripMenuItem_Click(object sender, Event
                 RestoreDirectory = true
             };
 
-            if (sfd.ShowDialog() == DialogResult.OK)
-            {
-                filename = sfd.FileName;
-            }
-            else
+            if (saveRomDialog.ShowDialog() != DialogResult.OK)
             {
                 return;
             }
 
+            var filename = saveRomDialog.FileName;
+
             RomParts rp = MemoryDomains.GetRomParts(currentSK.SystemName, currentSK.RomFilename);
 
             File.Copy(currentSK.RomFilename, filename, true);
-            using (FileStream output = new FileStream(filename, FileMode.Open))
+            using var output = new FileStream(filename, FileMode.Open);
```

Comment:
Is there a style rule that recommends this style? I far prefer the traditional C# syntax over the C#8 syntax as I think it provides much clearer scope as to the intention of the object

Reply:
I think it's an `IDE` rule (not `CA` or `SA`). I'll see if I can disable that rule in `.editorconfig` and revert this change

Reply:
If possible, I'd prefer that. While I like some of the more modern syntax, part of the point of using (in my eyes) is that it helps declare scope. It makes it abundantly clear that there's a resource in use, and prevents accidental misuse of that resource by forcing an extra scope.

Reply:
I prefer the old syntax too. I've been defaulting to resolving all Errors/Warnings/Infos so that we can fine-tune the `.editorconfig` file to reflect our style preferences.

Reply:
The latest commit has this change. I still need to resolve merge conflicts.